source/Cosmos.System2/Graphics/SVGAIICanvas.cs

```diff
@@ -360,7 +415,14 @@ public void CreateCursor()
         /// <exception cref="NotImplementedException">Thrown if VMWare SVGA 2 has no rectangle copy capability</exception>
         public void CopyPixel(int aX, int aY, int aNewX, int aNewY, int aWidth = 1, int aHeight = 1)
         {
-            _xSVGADriver.Copy((uint)aX, (uint)aY, (uint)aNewX, (uint)aNewY, (uint)aWidth, (uint)aHeight);
+            if (!DoubleBuffered)
```

Comment:
I think its easier if we just implemenent copy in the double buffered driver to remove the difference here.